rebuild_wrappers.py

```diff
@@ -1730,7 +1774,25 @@ def function_writer(f, N: FunctionType, W: str) -> None:
 			if k != str(Clauses()):
 				file.write("#endif\n")
 		file.write(files_guard["wrapper.h"].format(lbr="{", rbr="}", version=ver))
-	
+	#               E          v          c     w      i      I      C     W      u      U      f    d    D     l      L      p    V    O          S          N          H      P    A    x          X          Y          b
+	formats_map = ["(error)", "(error)", "i8", "i16", "i32", "i64", "u8", "u16", "u32", "u64", "f", "f", "Lf", "i64", "u64", "p", "p", "(error)", "(error)", "(error)", "u64", "p", "p", "(error)", "(error)", "(error)", "(error)"]
+
+	# Rewrite the x64printer.c file:
+	with open(os.path.join(root, "src", "emu", "x64printer.c"), 'w') as file:
+		file.write(files_header["x64printer.c"].format(lbr="{", rbr="}", version=ver))
+		
+		signature: RedirectType = ""
+		for signature in alltypedefs:
+			if any(letter in signature[2:] for letter in "OSNxXYb"):
+				continue
+			formats = signature[2:].replace("G", "UU")
+			params = '' if formats == "v" else ", " + function_args_systemV(formats)[:-2]
+			formats = '' if formats == "v" else ', '.join("%\" PRI{x} \"".format(x=formats_map[signature.get_convention().values.index(t)]) for t in formats)
+			file.write("    {rbr} else if ({condition}) {lbr}\n".format(condition=" || ".join(["!strcmp(func, \"" + func + "\")" for func in alltypedefs[signature]]), lbr="{", rbr="}"))
```

Comment:
Instead of comparing the function name, maybe try to compare the function called? So you compare `w` with `&iFv` instead of `s` (which you called `fnc` here) against every function name which takes no argument and return an int. That would also reduce the amount of instruction it adds (50M is a lot! I'd think a lot of this is simply storing the mapping as there are a _lot_ of functions.)

Reply:
yeah, that's why I converted this into a draft, working on it.